.travis.yml

```diff
@@ -68,7 +68,7 @@ before_install:
       export PHPUNIT=$(readlink -f ./phpunit)
       export PHPUNIT_X="$PHPUNIT --exclude-group tty,benchmark,intl-data"
       export COMPOSER_UP='composer update --no-progress --no-suggest --ansi'
-      export COMPONENTS=$(find src/Symfony -mindepth 2 -type f -name phpunit.xml.dist -printf '%h\n')
+      export COMPONENTS=$(find src/Symfony -mindepth 2 -type f -name phpunit.xml.dist -printf '%h\n' | sort)
```

Comment:
They're already sorted - but the output is in random order because tests run in parallel. we don't want to change that.

Reply:
The command `find src/Symfony -mindepth 2 -type f -name phpunit.xml.dist -printf '%h\n'` gave me an unsorted the output:
```
src/Symfony/Contracts
src/Symfony/Bundle/TwigBundle
src/Symfony/Bundle/WebServerBundle
src/Symfony/Bundle/WebProfilerBundle
src/Symfony/Bundle/SecurityBundle
src/Symfony/Bundle/FrameworkBundle
src/Symfony/Bundle/DebugBundle
src/Symfony/Bridge/ProxyManager
src/Symfony/Bridge/PhpUnit
src/Symfony/Bridge/Twig
src/Symfony/Bridge/Doctrine
src/Symfony/Bridge/Monolog
src/Symfony/Component/Cache
src/Symfony/Component/Translation
src/Symfony/Component/Security/Core
src/Symfony/Component/Security/Http
src/Symfony/Component/Security
src/Symfony/Component/Security/Guard
src/Symfony/Component/Security/Csrf
src/Symfony/Component/HttpKernel
src/Symfony/Component/Yaml
src/Symfony/Component/OptionsResolver
src/Symfony/Component/Config
src/Symfony/Component/PropertyAccess
src/Symfony/Component/Mime
src/Symfony/Component/WebLink
src/Symfony/Component/HttpFoundation
src/Symfony/Component/Lock
src/Symfony/Component/Serializer
src/Symfony/Component/Process
src/Symfony/Component/Console
src/Symfony/Component/EventDispatcher
src/Symfony/Component/Asset
src/Symfony/Component/ExpressionLanguage
src/Symfony/Component/Workflow
src/Symfony/Component/Ldap
src/Symfony/Component/BrowserKit
src/Symfony/Component/PropertyInfo
src/Symfony/Component/VarDumper
src/Symfony/Component/Stopwatch
src/Symfony/Component/Routing
src/Symfony/Component/Dotenv
src/Symfony/Component/Debug
src/Symfony/Component/Inflector
src/Symfony/Component/Templating
src/Symfony/Component/DomCrawler
src/Symfony/Component/DependencyInjection
src/Symfony/Component/Form
src/Symfony/Component/Filesystem
src/Symfony/Component/Intl
src/Symfony/Component/CssSelector
src/Symfony/Component/Messenger
src/Symfony/Component/HttpClient
src/Symfony/Component/VarExporter
src/Symfony/Component/Finder
src/Symfony/Component/Validator
```

But I didn't know that parallelization mess it up too :)

Reply:
sorting is left to the filesystem then :)
but since parallelization changes the order anyway, not sure this is worth it

Reply:
Not worth it as it give partial results